Assets/Scripts/GameManager.cs

```diff
@@ -6,7 +6,7 @@
 
 public class GameManager : MonoBehaviour {
     public static GameManager instance;
-    public bool gameOver = false;
+    public bool GameOver;
```

Comment:
Should this remain `public`? This enables someone to start the game with the `GameOver` field set as true.